lib/util/fields/password_field.dart

```diff
@@ -1,39 +1,114 @@
 import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
 
-class PasswordField extends StatelessWidget {
-  final String labelText;
-  final String hintText;
-  final String? errorText;
-  final String? helperText;
-  final String? Function(String?)? validator;
-  final TextEditingController? controller;
+class PasswordField extends StatefulWidget {
+  static const Key passwordFieldKey = Key('passwordField');
+  static const Key passwordConfirmationFieldKey = Key('passwordConfirmationField');
+
+  final TextEditingController controller;
+  final TextEditingController? originalPasswordController;
```

Comment:
I don't think this is a good name. I don't have any GREAT ideas for alternatives, but this sounds to me like there is an "original" and an "actual" password controller for the same field

Reply:
I thought about `passwordForConfirmationController` which would be accurate, but very confusing because it sounds like the exact opposite of what it does. I can definitely add a comment here, though.